conan/tools/cmake/toolchain/blocks.py

```diff
@@ -112,6 +112,53 @@ def context(self):
 
         return {"vs_runtimes": config_dict}
 
+class VSDebuggerEnvironment(Block):
+    template = textwrap.dedent("""
+        # Definition of CMAKE_VS_DEBUGGER_ENVIRONMENT
+        {% if vs_debugger_path %}
+        {% set genexpr = namespace(str='') %}
+        {% for config, value in vs_debugger_path.items() %}
+        {% set genexpr.str = genexpr.str +
+                                '$<$<CONFIG:' + config + '>:' + value|string + '>' %}
```

Comment:
I prefer this logic in Python, is easier to maintain and debug IMHO (but good for me if you want to keep it)

Reply:
Done!